src/universal/generate-random-pkmn.c

```diff
@@ -0,0 +1,155 @@
+#include <pksm.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <time.h> 
+#include <string.h>
+
+int main(int argc, char **argv)
+{
+    unsigned char version = *argv[2];
+    enum Generation gen;
+
+    int maxSpecies = 0;
+    int maxMoves = 0;
+    int maxBalls = 0;
+    int maxItems = 0;
+    int maxAbility = 0;
+
+    gui_warn("Please enter how many pokemon", "you would like to generate!");
+
+    int randomizedCount = 0;
+    gui_numpad(&randomizedCount, "You are entering how many randomized\npokemon you would like to generate", 3);
```

Comment:
This is a much longer "What?" message than normal. Second-person is also kind of odd here, in my opinion

Reply:
fixed